src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -132,9 +130,12 @@ public static final class GoalPos {
           new GoalPos(0, 0)
         };
         public static GoalPos[] INTAKE = {
-          new GoalPos(Units.degreesToRadians(-72.5), Units.degreesToRadians(73.37)),
-          new GoalPos(Units.degreesToRadians(-66.6), Units.degreesToRadians(29.94)) 
+          new GoalPos(Units.degreesToRadians(-72.5), Units.degreesToRadians(92.21)),
+          new GoalPos(Units.degreesToRadians(-66.6), Units.degreesToRadians(48.78)) 
         };
+        // Rollers are asymmetric, so when moving rollers to other side of robot, this value will correct
+        // the roller so that the rollers will act as if its on the front side
+        public static double asymmetricCorrectionDeg = 10;
```

Comment:
Unfortunately, it's not that simple. Consider the case of placing a cube in the front instead of the back.